src/components/hero-section/HeroSection.vue

```diff
@@ -249,6 +287,62 @@ export default {
             }
         }
 
+        &__video-wrapper {
+            grid-row: 1;
+            grid-column: 1 / -1;
+            position: relative;
+            line-height: 0;
+
+            .vs-hero-section__video {
+                width: 100%;
+                height: 560px;
+                object-fit: cover;
+
+                @include media-breakpoint-up(sm) {
+                    height: 648px;
+                }
+                @include media-breakpoint-up(lg) {
+                    height: 812px;
```

Comment:
These heights feel very specific, it's not an issue and it looks good  in practice, I just wondered where they're derived from?

Reply:
I'm not sure, they just follow the designs. I think its probably just ensuring the height doesn't go below the fold